### ISSUE TITLE
Dashboard Down

### ISSUE THREAD

**Issue by dzlandis** [openeew/openeew#30]
It appears that https://dashboard.openeew.com/ is down. I am unable to access the website.

**Comment by Tropaion** [comment]
Yes, sadly it looks like the project is dead. Just wanted to build an OpenEEW sensor
